database/factories/CourseFactory.php

```diff
@@ -61,9 +62,9 @@ public function configure()
             $filename_image = "course/$id/" . $word . ".jpg";
             $filename_video = "course/$id/" . $word . ".mp4";
             $filename_poster = "course/$id/" . $word . "poster.jpg";
-            $dest_image = storage_path("app/public/$filename_image");
-            $dest_video = storage_path("app/public/$filename_video");
-            $dest_poster = storage_path("app/public/$filename_poster");
+            $dest_image = Storage::path("app/public/$filename_image");
+            $dest_video = Storage::path("app/public/$filename_video");
+            $dest_poster = Storage::path("app/public/$filename_poster");
```

Comment:
```suggestion
            $dest_image = Storage::path("app/public/$filename_image");
            $dest_video = Storage::path("app/public/$filename_video");
            $dest_poster = Storage::path("app/public/$filename_poster");
```

not sure about this but 

it we use default `disk` then it use `storage/app` by default 


so .... 

```suggestion
            $dest_image = Storage::path("$filename_image");
            $dest_video = Storage::path("$filename_video");
            $dest_poster = Storage::path("$filename_poster");
```

or 

```suggestion
            $dest_image = Storage::path("public/$filename_image");
            $dest_video = Storage::path("public/$filename_video");
            $dest_poster = Storage::path("public/$filename_poster");
```

![image](https://user-images.githubusercontent.com/214608/147221994-52ab74b3-bf3b-48dd-9c43-12dea314cf05.png)

@pa-cholek @mako321 reflections ?

Reply:
It must be either `Storage::disk('public')->path($filename_image)` or `Storage::path("public/$filename_image")`, if we want these files accessible by URL.

Reply:
First one is better though, because it does not depend on what is set as `default` disk / or if there were some changes to how `local` disk is defined. 
For example our API has `local` disk set to same config as `public`, so doing Storage::path('public/$filename_image') would create path of '/app/public/public/xxx'...
 
But maybe we should add Configuration for which disk we want to use for Courses package anyway?

Reply:
`Storage::path` uses `default` disk, that can be changed in app/cofigs/* right ? 

Lets stay with that, but we could setup default disk in `core`